File: examples/codespace/NodeJS/connect.js
var Connection = require('tedious').Connection;  
    
var config = {
    server: '<your_server.database.windows.net>',
    authentication: {
        type: 'default',
        options: {
        userName: '<your_username>',
        password: '<your_password>' 
        }
    },
    options: {
        database: 'master',
        port: <your_database_port>,
        trustServerCertificate: true,
        encrypt: true
    }
};

const connection = new Connection(config);

connection.connect((err) => {  
        if (err) 
        {
            console.log(err)
        }else{
            console.log("Connected");
            connection.close();
        }
});

File: examples/codespace/Python/connect.py
import pyodbc 

server = 'tcp:<your_server.database.windows.net>'
database = 'master'
username = '<your_username>'
password = '<your_password>'
port = '<your_port_number>'

# ENCRYPT defaults to yes starting in ODBC Driver 18. It's good to always specify ENCRYPT=yes on the client side to avoid MITM attacks.
cnxn = pyodbc.connect('DRIVER={ODBC Driver 18 for SQL Server};SERVER='+server+','+port+';DATABASE='+database+';ENCRYPT=yes;TrustServerCertificate=yes;UID='+username+';PWD='+ password)
cursor = cnxn.cursor()

#Sample select query
cursor.execute("SELECT @@version;") 
row = cursor.fetchone() 
while row: 
    print(row[0])
    row = cursor.fetchone()